cpp/libs/gst_nmos_audio_receiver_plugin/utils.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+#include <string>
+#include <glib.h>
+#include <variant>
+#include <gst/gst.h>
+#include <nlohmann/json.hpp>
+
+typedef struct node_fields_t
```

Comment:
If this .h file is meant for C++, then you don't need the `typedef struct`, you can just use `struct`.